tendrl/node_agent/monitoring/collectd/collectors/gluster/tendrl_gluster_brick_disk_stats.py

```diff
@@ -542,10 +542,18 @@ def get_metrics(self):
                 []
             ).iteritems():
                 for brick in sub_volume_bricks:
+                    brick_hostname = gluster_utils.find_brick_host(
+                        self.etcd_client, brick['hostname']
+                    )
+                    if brick_hostname:
+                        brick_ip = socket.gethostbyname(brick_hostname)
+                    else:
+                        brick_ip = socket.gethostbyname(brick['hostname'])
+                        brick_hostname = brick['hostname']
```

Comment:
what is the purpose of L#553?   , you are simply assigning new value to `brick_ip", the why the if/else above